optuna/trial/_trial.py

```diff
@@ -396,6 +396,10 @@ def objective(trial):
 
         Returns:
             A suggested value.
+        
+        .. seealso::
+            :ref:`configurations` tutorial describes more details and flexible usages.
+        
```

Comment:
```suggestion

```

Reply:
Both of these change mean deleting a line?

Reply:
Oh, sorry. I didn't notice this comment. No. I mean that it would be great to delete the spaces, not the whole line.

Reply:
I've left comments on this PR about this. Could you look at them?